ndk-build/src/apk.rs

```diff
@@ -85,11 +85,11 @@ impl<'a> UnalignedApk<'a> {
         std::fs::copy(path, out.join(&file_name))?;
 
         let mut aapt = self.0.build_tool(bin!("aapt"))?;
-        aapt.arg("add").arg(self.0.unaligned_apk()).arg(format!(
-            "lib/{}/{}",
-            abi,
-            file_name.to_str().unwrap()
-        ));
+        aapt.arg("add")
+            .arg("-0")
```

Comment:
`-0 ""` disables compression for any file extension.

Reply:
This should be rather added to the `aapt package` command

Reply:
@msiglreith I think it should be on both, as the argument only has effect on the files added through that particular command. We don't add any (e.g. heavy `.so`) files through `aapt package`, only through `aapt add`.